src/main/java/com/google/devtools/build/lib/rules/apple/AppleCommandLineOptions.java

```diff
@@ -82,6 +82,17 @@ public class AppleCommandLineOptions extends FragmentOptions {
               + "If unspecified, uses default iOS SDK version from 'xcode_version'.")
   public DottedVersion.Option iosSdkVersion;
 
+  @Option(
+      name = "visionos_sdk_version",
+      defaultValue = "null",
+      converter = DottedVersionConverter.class,
+      documentationCategory = OptionDocumentationCategory.TOOLCHAIN,
+      effectTags = {OptionEffectTag.LOSES_INCREMENTAL_STATE},
+      help =
+          "Specifies the version of the visionOS SDK to use to build visionOS applications. "
+              + "If unspecified, uses default visionOS SDK version from 'xcode_version'.")
+  public DottedVersion.Option visionOsSdkVersion;
```

Comment:
I removed this one and just always default to the version discovered from Xcode. I'm actually not sure what the `*sdk_version` flags are for. Since Xcode selection should dictate this instead.